staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -156,19 +156,29 @@ func (rl *respLogger) Addf(format string, data ...interface{}) {
 // Log is intended to be called once at the end of your request handler, via defer
 func (rl *respLogger) Log() {
```

Comment:
Please remove `Log()` function and replace calls to it with 
```
klog.V(3).InfoS("Recieved HTTP request", rl.LogArgs()...)
```

Reply:
I intended to keep the source same since Log function may have been used somewhere else as well

Reply:
We should replace all usage of Log function. This is due InfoDepth is no longer available.

Reply:
yes but isn't that replaced all together if we keep the wrapper same?

Reply:
I'm not sure I understand. Please just replace Log() as described in migration doc